src/main/java/tools/mapletools/CashDropFetcher.java

```diff
@@ -260,7 +262,8 @@ private static void reportNxDropResults(boolean dropdata) throws SQLException {
     }
 
     private static void reportNxDropData() {
-        try {
+        //NEED FUTURE UPDATE
```

Comment:
Why does it need a future update?

Reply:
I haven't implements Java NIO on this file